stack-graphs/src/graph.rs

```diff
@@ -426,8 +426,8 @@ impl Handle<File> {
 #[repr(C)]
 #[derive(Clone, Copy, Default, Eq, Hash, Ord, PartialEq, PartialOrd)]
 pub struct NodeID {
-    file: ControlledOption<Handle<File>>,
-    local_id: u32,
+    pub(crate) file: ControlledOption<Handle<File>>,
+    pub(crate) local_id: u32,
```

Comment:
There are public `file` and `local_id` accessor methods that you should use instead of publicizing these fields.

Reply:
The problem is that `local_id()` cannot be called when `file` is absent. I did this to avoid changing the public API. Which of the following options would yuo prefer:
1. Keep as-is.
2. Drop the assertion from `local_id()`, so that it can be called on `root` and `jump_to_scope` nodes.
3. Make `ROOT_NODE_ID` and `JUMP_TO_NODE_ID` crate visible and use those directly in the serializer.
4. Hard code the node ids in the serializer. :trollface:

Reply:
I went with (2) for now. If you feel it should be one of the other options, I'll open another patch PR for that.